File: FastClass/salida/java/test4.java
public class test4 { 
	String property;
	int read;
	int write;
	int const;

	public test4(String property, int const) {
		this.property = property;
		this.const = const;
	}
	public String getProperty(){
		return this.property;
	}
	public void setProperty(String property){
		this.property = property;
	}
	public int getRead(){
		return this.read;
	}
	public void setWrite(int write){
		this.write = write;
	}

}

File: FastClass/salida/php/test4.php
<?php 
class test4 { 
	public $property;
	public $read;
	public $write;
	public $const;

	public __construct($property, $const) {
		$this->property = $property;
		$this->const = $const;
	}
	public function getProperty(){
		return $this->property;
	}
	public function setProperty($property){
		$this->property = $property;
	}
	public function getRead(){
		return $this->read;
	}
	public function setWrite($write){
		$this->write = $write;
	}

}
?>
